### ISSUE TITLE
Support tf.IsFinite lowering

### ISSUE THREAD

**Issue by WindQAQ** [tensorflow/tensorflow#47361]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Colab
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version, if pip package or github SHA, if built from source): nightly

### 2. Code

```python
import tensorflow as tf

@tf.function
def func(x):
  return tf.is_finite(x)

converter = tf.lite.TFLiteConverter.from_concrete_functions([func.get_concrete_function(tf.TensorSpec(shape=(1, 2, 3)))])
converter.convert()
```

### 3. Failure after conversion

It will fail if flex is not enabled. However, `tf.is_inf()` is supported without flex because it is lowered to other ops in `lower_tf.cc`. It should be easy to support lowering pattern of `tf.is_finite`. Note that `tf.is_finite` is also being used in `tf.keras.layers.Softmax` with multi-axes reduction. I understand that I can enable flex to use it, but it should be a minimum work to lower it directly.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/advanced_activations.py#L328-L345

**Comment by abattery** [comment]
Thanks for filing the feature request! Is there any plan for this in your side?

**Comment by thaink** [comment]
@WindQAQ I'll make a quick PR to support op lowering.

**Comment by WindQAQ** [comment]
> Thanks for filing the feature request! Is there any plan for this in your side?

@abattery Is probably like `tf.logical_not(tf.logical_or(tf.is_nan(x), tf.is_infinite(x)))`

> @WindQAQ I'll make a quick PR to support op lowering.

Thank you for the help!

**Comment by thaink** [comment]
Finally my PR is merged at e89316dfca814d2d2ff2bef47d33bc33140cc112
The tf.IsFinite is lowered to tf.Equal(tf.Subtract(x, x), 0).
